src/coreclr/vm/interoplibinterface_shared.cpp

```diff
@@ -154,3 +154,142 @@ void Interop::OnAfterGCScanRoots(_In_ bool isConcurrent)
         ObjCMarshalNative::AfterRefCountedHandleCallbacks();
 #endif // FEATURE_OBJCMARSHAL
 }
+
+#ifdef FEATURE_GCBRIDGE
+
+namespace
+{
+    Volatile<BOOL> g_GCBridgeActive = FALSE;
```

Comment:
```suggestion
    Volatile<bool> g_GCBridgeActive = false;
```
Nit: It is fine to use native C/C++ bool for internal details.